mariadb/data/run.sh

```diff
@@ -2,18 +2,24 @@
 set -e
 
 MARIADB_DATA=/data/databases
+NEW_INSTALL=false
 
 # Init mariadb
 if ! bashio::fs.directory_exists "${MARIADB_DATA}"; then
     bashio::log.info "Create a new mariadb initial system"
-    mysql_install_db --user=root --datadir="$MARIADB_DATA" > /dev/null
+    mysql_install_db --user=root --datadir="${MARIADB_DATA}" --skip-name-resolve --skip-test-db > /dev/null
+    NEW_INSTALL=true
 else
     bashio::log.info "Using existing mariadb initial system"
 fi
 
+# Redirect log output
+rm -f /data/databases/local-mariadb.err
+ln -s /proc/1/fd/1 /data/databases/local-mariadb.err
```

Comment:
This is not good for production I now realize, we might need to fix this in the mysql cnf by making it more predictable using the `log_error` directive.

Reply:
Adjusted.